shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -463,7 +463,17 @@ - (BOOL)flt_hasFirstResponderInViewHierarchySubtree {
 void FlutterPlatformViewsController::CompositeWithParams(int view_id,
                                                          const EmbeddedViewParams& params) {
   CGRect frame = CGRectMake(0, 0, params.sizePoints().width(), params.sizePoints().height());
-  UIView* touchInterceptor = touch_interceptors_[view_id].get();
+  FlutterTouchInterceptingView* touchInterceptor = touch_interceptors_[view_id].get();
+  FML_DCHECK(CGPointEqualToPoint([touchInterceptor embeddedView].frame.origin, CGPointZero));
+  if (!CGPointEqualToPoint([touchInterceptor embeddedView].frame.origin, CGPointZero)) {
+    FML_LOG(WARNING)
+        << "Embedded PlatformView's origin is not CGPointZero."
+           "A none-zero origin might cause undefined behavior."
```

Comment:
> Do we want them to file a new issue or to add a comment to the existing issue?

This is a good point, ill instead ask them to comment on the issue. 

> How does an application developer fix their code if they see this warning?

There is probably nothing they can do. It would be the same behavior they have been had. They will just have to ignore this error message. (It can be annoying seeing this every frame tho, I can add some code to only print this once. I'm not sure how to test this tho)

> We should at least log the view_id, which hopefully is enough for them to figure out which view it is - if it's possible to tell them something about the type that would be good too (e.g. if an application has multiple platform views).

Good point, will do.

Reply:
How about we ifdef this out of release and profile builds, and just keep a set of view_ids - if the set contains the ID don't print the message, otherwise insert it and print.

Reply:
We should also make sure that this log level gets printed from the command line in an opt build without specifying `--verbose-system-logs`.